template-for-repository/proofs/Makefile.common

```diff
@@ -351,6 +351,14 @@ CBMC_USE_FUNCTION_CONTRACTS := $(patsubst %,--replace-call-with-contract %, $(US
 # the APPLY_LOOP_CONTRACTS variable to 1.
 APPLY_LOOP_CONTRACTS ?= 0
 
+# Set VERBOSE=1 to see the full Makefile output, but silence by default
+# the noisy output (for example, the long `litani add-job` invocations)
+ifdef VERBOSE
+  QUIET =
```

Comment:
Instead of decorating every single recipe, why not just pass the `-q` flag to Make?

```
ifndef VERBOSE
    MAKEFLAGS := $(MAKEFLAGS) -q
endif
```
I think that should be sufficient

Reply:
Done